tools/ci_build/github/apple/c/assemble_c_pod_package.py

```diff
@@ -50,6 +52,8 @@ def main():
     # copy the necessary files to the staging directory
     framework_dir = args.framework_dir.resolve()
     shutil.copytree(framework_dir, staging_dir / framework_dir.name, dirs_exist_ok=True)
+    public_headers_dir = args.public_headers_dir.resolve()
+    shutil.copytree(public_headers_dir, staging_dir / public_headers_dir.name, dirs_exist_ok=True)
     copy_repo_relative_to_dir(["LICENSE"], staging_dir)
```

Comment:
>LICENSE [](http://example.com/codeflow?start=32&length=7)

do we need to copy the license file here if it's also copied to the xcframework in build_ios_framework.py?

Reply:
Actually we don't need to copy in multiple steps, we can copy everything from the framework_out directly, since it contains everything we need here,
Can change in a later PR